ara/alert-docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+version: "3.5"
+services:
+
+  elastalert:
+    container_name: elastalert
+    image: kingstonduo/elastalert
+    volumes:
+      - ${ENV_ELASTALERT_CONF}:/opt/elastalert/config.yaml
+      - ${ENV_ELASTALERT_TEST_CONF}:/opt/elastalert/config-test.yaml
+      - ${ENV_ELASTALERT_SERVER_CONF}:/opt/elastalert-server/config/config.json
+      - ${ENV_ELASTALERT_RULES_DIR}:/opt/elastalert/rules
+      - ${ENV_ELASTALERT_RULES_TEMPLATE_DIR}:/opt/elastalert/rule_templates
+    logging:
+      options:
+        max-size: "10m"
+        max-file: "3"
+    networks:
+      local:
+        aliases:
+          - elastalert
+
+networks:
```

Comment:
I think that the main docker-compose needs to own the network, and this one needs to join it.

The code to join a network looks like this, where "dataLake" is the name of the existing network to join:
```
networks:
  dataLake:
    external: true
```

Reply:
In order for this to work, we need to set up a network named logAggregator in the other docker-compose file that we can join.

Reply:
We do have one, is there something I'm missing that needs to be on it to open it up for external connections??


`networks:
  local:
    name: logAggregator
    driver: bridge
    ipam:
      driver: default
      config:
        - 
          subnet: 10.3.0.0/16`

Reply:
Oh, that that didn't paste in very nicely, but see bottom of the the docker-compose.yml file

Reply:
Ok...I was thinking that didn't exist yet, and didn't see it as a change. Thanks